lib/linalg/svd/IncrementalSVDFastUpdate.cpp

```diff
@@ -136,9 +148,111 @@ IncrementalSVDFastUpdate::buildInitialSVD(
     d_num_samples = 1;
     d_num_rows_of_W = 1;
 
-    // Compute the basis vectors for this time interval.
-    computeBasis();
+}
+
+bool
+IncrementalSVDFastUpdate::buildIncrementalSVD(
```

Comment:
I believe this should be mathematically equivalent to the previous version, i.e. `IncrementalSVD::buildIncrementalSVD`. If so, can we add a unit test that checks the equivalence?